packages/mobile/src/escrow/saga.ts

```diff
@@ -1,7 +1,6 @@
+import { EscrowWrapper } from '@celo/contractkit/lib/wrappers/Escrow'
+import { StableTokenWrapper } from '@celo/contractkit/lib/wrappers/StableTokenWrapper'
```

Comment:
Why can't we just import from `@celo/contractkit`?

Reply:
It's not exported from the index file